product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -26,27 +26,28 @@ Upon starting, PGD Proxy connects to one of the endpoints given in the local con
+PGD Proxy provides the consensus_grace_period proxy option that can be used to configure the routing behavior upon loss of a Raft leader. PGD Proxy continues to route to the current write leader (if it's available) for this duration. If the new Raft leader isn't elected during this period, the proxy stops routing. If set to `0s`, PGD Proxy stops routing immediately.
 
-The main purpose of this option is to allow users to configure the write behaviour when the Raft leader is lost. When the Raft leader is not present in the cluster it is not always guaranteed that the current write leader seen by the proxy is the correct one. In some cases, like network partition in the following example, it is possible that the two write leaders may be seen by two different proxies attached to the same group increasing the chances of write conflicts. If this is not the desired behavior then the previously mentioned  consensus_grace_period can be set to 0s. This setting configures the proxy to stop routing and close existing open connections immediately when it detects the Raft leader is lost.
+The main purpose of this option is to allow users to configure the write behavior when the Raft leader is lost. When the Raft leader isn't present in the cluster, it's not always guaranteed that the current write leader seen by the proxy is the correct one. In some cases, like network partition in the following example, it is possible that the two write leaders may be seen by two different proxies attached to the same group increasing the chances of write conflicts. If this isn't the desired behavior, then the previously mentioned `consensus_grace_period` can be set to 0s. This setting configures the proxy to stop routing and closes existing open connections immediately when it detects the Raft leader is lost.
 
 #### Network partition example
 
-Consider a 3 data node group with a proxy on each data node. In this case, if the current write leader gets network partitioned or isolated then the data nodes present in the majority partition will elect a new write leader. If `consensus_grace_period` is set to a non-zero value say `10s` then the proxy present on the previous write leader will continue to route writes for this duration.
+Consider a 3-data node group with a proxy on each data node. In this case, if the current write leader gets network partitioned or isolated, then the data nodes present in the majority partition elects a new write leader. If `consensus_grace_period` is set to a non-zero value, say `10s`, then the proxy present on the previous write leader continues to route writes for this duration.
 
-Note: In this case, if the grace period is kept too high then writes will continue to happen on the two write leaders increasing the chances of write conflicts.
+!!! Note
+    In this case, if the grace period is kept too high, then writes continue to happen on the two write leaders. This condition increases the chances of write conflicts.
```

Comment:
The item in question doesn't really rise to needing an admonition, just better wording.

```suggestion
Note that in this case, if the grace period is kept too high, then writes continue to happen on the two write leaders. This condition increases the chances of write conflicts.
```